cluster/update-storage-objects.sh

```diff
@@ -59,7 +61,12 @@ for resource in "${resources[@]}"
 do
   for namespace in "${namespaces[@]}"
   do
+    # If get fails, assume it's because the resource hasn't been installed in the apiserver.
+    # TODO hopefully we can remove this once we use dynamic discovery of gettable/updateable
+    # resources.
+    set +e
```

Comment:
Yup. Is it possible to differentiate between getting 0 resources and getting an error?
I want that there should be atleast one resource for which this command does not return an error. Returning 0 instances is fine.